### ISSUE TITLE
wxWidgets interface support

### ISSUE THREAD

**Issue by asmwarrior** [NanoMichael/cLaTeXMath#30]
Hi, I see there are some development of QT interface, but is it possible to support the wxWidgets?

There are some discussion here: [How to display math formula or symbol in wxWidgets?](https://forums.wxwidgets.org/viewtopic.php?f=1&t=46948)

Any ideas? Thanks

**Comment by NanoMichael** [comment]
Yes, basically you need to warp the wxWidgets graphical backend to implement the interfaces declared in the file `src/graphic/graphic.h` (see the demo app in `src/platform/cairo/*` or `src/platform/qt/*`), and you can check the demo app `src/samples/gtkmm_main.cpp` to see how it works.
